File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/C+DS/codes/Recursion/p21.c
// count zero..


#include<stdio.h>
int countless(int num){
	int count=0;
	if(num==0){
		return num;
	}
	while(num){
		if(num%10==0){
			count++;
			num=num/10;
		}
		return count;
	}
}
void main(){
	int num;
	printf("Enter num to search\n");
	scanf("%d",&num);
	printf("%d\n",num);
}

File: java-main/java-main/Nestedforloop/p21.java
/*
      
    A 1 B 2 
    C 3 D
    E 4 
    F
    */
class Demo{

        public static void main(String[]args){

                int N=4;
		char ch='A';
		int num=1;

                for(int i=1;i<=N;i++){
                        for(int j=1;j<=N-i+1;j++){
                                if(j%2==1){
                                System.out.print(ch++ + " ");
                                }else{
                                        System.out.print(num++ + " ");
			}
		}
		System.out.println();
		}
	}
}
              

